dom.bs

```diff
@@ -660,7 +660,7 @@ flags that are all initially unset:
 <ul>
  <li><dfn export for=Event id=stop-propagation-flag>stop propagation flag</dfn>
  <li><dfn export for=Event id=stop-immediate-propagation-flag>stop immediate propagation flag</dfn>
- <li><dfn export for=Event id=canceled-flag>canceled flag</dfn>
+ <li><dfn export for=Event id=canceled-flag>aborted flag</dfn>
```

Comment:
This and similar changes to Events will need to be reverted. Simple search & replace is too simple unfortunately.